--- node-phantom.js	2013-10-18 22:20:46.000000000 +0300
+++ node-phantom.patched.js	2016-06-10 12:32:33.023939163 +0300
@@ -48,7 +48,7 @@
 			response.writeHead(200,{"Content-Type": "text/html"});
 			response.end('<html><head><script src="/socket.io/socket.io.js" type="text/javascript"></script><script type="text/javascript">\n\
 				window.onload=function(){\n\
-					var socket = new io.connect("http://" + window.location.hostname);\n\
+					var socket = new io.connect("http://" + window.location.host);\n\
 					socket.on("cmd", function(msg){\n\
 						alert(msg);\n\
 					});\n\
